CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,4 @@
-# ngrok Documentation Code of Conduct
```

Comment:
lol, not surprised that this has said ngrok docs for so long
no one reads it and i def stole it from there haha